api-lib/validate.ts

```diff
@@ -18,6 +34,7 @@ export const verifyHasuraRequestMiddleware = (handler: VercelApiHandler) => {
     try {
       await handler(req, res);
     } catch (error: any) {
+      Sentry.captureException(error);
```

Comment:
We're just going to only capture unhandled exceptions that throw all the way up? I think that makes a lot of sense, since it'll have a really high signal-to-noise ratio, but I'm just a tad concerned about false negatives falling through the cracks.

Reply:
IMO, since this is all pretty new and untested in prod, it's worth starting out noisier and dialing it down afterward.

Reply:
I changed it to capture all error responses